bibliopixel/main/all_pixel.py

```diff
@@ -23,11 +23,11 @@
 ]
 
 
-def O(msg):
+def O(msg):  # noqa: E743
```

Comment:
This doesn't actually work in flake8 last I checked, it will instead disable flake8 for the entire file.

Reply:
Wait... actually, check your version of flake8... 3.1 and up should work.

Reply:
Eh, it's not an issue as I'll get rid of those entirely in a subsequent change.